styles/_elements.scss

```diff
@@ -570,8 +570,12 @@
     }
   }
 
-  // leaving blank for now as class exists and should be seen as required for future use
   .sectionContent {
-  }
-}
-
+    padding: var(--space-4) var(--space-6);
+    margin-bottom: var(--space-4);
+    border-radius: var(--card-border-radius);
+    background: var(--bg-white);
+    box-shadow: var(--card-shadow);
+    border: 1px solid var(--card-border);
+  }
+}
```

Comment:
Added a style for the `sectionContent` class. It was taken from the existing templateListItem style.

Reply:
I still feel that the sections and section content should make use of the containers. New containers can be created using those as a basis,

So `<LayoutWithSections>`, inheriting from `LayoutContainer`, and `<SectionContent>` inheriting from `ContentContainer`.

Reply:
@andrewebdev I didn't see a `<SectionContent>` container, so I used the `sectionContent` class in the User Profile page to break out the `Profile` part from the `Email addresses`.

Reply:
Just as an example, both the `User Profile` and `Manage Access` pages use `sectionContainer` with `sectionContent` because there are separate sections within the same page:
![Uploading image.png…]()


But the Manage Access page was originally created without the white boxes around the sections, but the User Profile page has those boxes around the sections.

So my understanding was that all sections needed to have a white background, and I added the above style to do that. Please provide guidance if that is not what you intended.

Reply:
I discussed with @fraserclark in our weekly meeting, and he said it's ok for now to set the style for "sectionContent" with a white background for the above cases.